packages/amesos2/src/Amesos2_PardisoMKL.cpp

```diff
@@ -307,7 +305,7 @@ TPETRA_ETI_MANGLING_TYPEDEFS()
   #endif
 #endif
 #ifdef HAVE_TPETRA_INST_COMPLEX_DOUBLE
-    #ifdef HAVE_TPETRA_INST_INT_INT 
```

Comment:
@mhoemmen looks like you missed removing the corresponding `#endif` here

Reply:
Thanks @ndellingwood !  I'll work on this.

Reply:
@ndellingwood I think this one looks fine -- thanks for checking, though!